### DISCUSSION TITLE
RustServer random seed not random

### DISCUSSION THREAD

**Discussion by YouMustNot** [GameServerManagers/LinuxGSM#3311] [Q&A] [answered]
It seems the random seed bug is still here?
I've left the seed empty but it only loads map seed 1 with wipe or wipeall

anyone else having this issue?

**Comment by YouMustNot** [comment]
no one else having this issue or is it just me?

**Reply by dgibbs64** [reply]
I am working on a hotfix and reworking the wipe command

**Comment by dgibbs64** [comment, marked as answer]
Hotfix is now out
